### ISSUE TITLE
Unc0ver Crashing After Attempted Jailbreak

### ISSUE THREAD

**Issue by VMT-Batman** [pwn20wndstuff/Undecimus#2092]
**Describe the bug**
After restarting, my iPhone 11 on 13.5 is not able to jailbreak. The app attempts to jailbreak, and then crashes without completing it. The app continues to crash unless rebooted

**To Reproduce**
1. Open Unc0ver App
2. Click on 'Jailbreak'
3. Wait
4. It'll get to about the 28th step and then just crash

**Screenshots**
Before restoring Rootfs: https://youtu.be/bqbA07wN7LM

After restoring Rootfs: https://youtu.be/6WUY6f-MEc8

**Device (please complete the following information):**
 - iOS Version: 13.5
 - iOS Device: iPhone 11
 - unc0ver Version: Tested on 5.0.1, 5.2.0, 5.2.1

**Place an "x" between the brackets if true:**
 - [ ] this is a bug others will be able to reproduce
 - [x] this issue is present with all tweaks uninstalled(except for default packages) or disabled
 - [x] this issue is present after a rootfs restore
 - [x] this issue is present on the latest version of unc0ver

**Logs**
If applicable, add logs or error messages here.
Present in videos linked

**Comment by Hamlock-maneuverr** [comment]
go to settings, general, reset all settings.  When you go to jailbreak, restore root FS first.  then reboot and try again.

**Comment by Hamlock-maneuverr** [comment]
You might also want to use a different cord when installing unc0ver.

**Comment by VMT-Batman** [comment]
> go to settings, general, reset all settings. When you go to jailbreak, restore root FS first. then reboot and try again.

Restoring settings didn't work unfortunately 

> You might also want to use a different cord when installing unc0ver.

What do you mean a cord? I went to the website and used the "Open in AltStore" option and for the older releases, I had them in my Files

**Comment by Hamlock-maneuverr** [comment]
If you look closely in that video at the 25 second mark.  you can see that you have an unauthenticated package "substituted"   ..   Also you left Gump installed before you did all of this.  That might be whats causing all this.

You need a high level programmer to help you with this.

**Comment by VMT-Batman** [comment]
> If you look closely in that video at the 25 second mark. you can see that you have an unauthenticated package "substituted" .. Also you left Gump installed before you did all of this. That might be whats causing all this.
> 
> You need a high level programmer to help you with this.

I see, thank you so much for this information. Theoretically, a factory reset should fix this right?

**Comment by Hamlock-maneuverr** [comment]
totally. But you will lose your jailbreak.  I think if you just run it normally (unjailbroken) until more information can be found about this.  I'm very confident it can easily be fixed.

**Comment by Hamlock-maneuverr** [comment]
just curious.  have you tried jailbreaking with "load tweaks" unchecked?

**Comment by VMT-Batman** [comment]
> just curious. have you tried jailbreaking with "load tweaks" unchecked?

Just tried it now and the same thing happened

**Comment by Hamlock-maneuverr** [comment]
You tried redownloading unc0ver yet?

**Comment by VMT-Batman** [comment]
> You tried redownloading unc0ver yet?

yeah, I've tried redownloading as well as using older versions but with no luck

**Comment by VMT-Batman** [comment]
> totally. But you will lose your jailbreak. I think if you just run it normally (unjailbroken) until more information can be found about this. I'm very confident it can easily be fixed.

Update on this. I tried Factory Resetting, and even that didn't work

**Comment by Hamlock-maneuverr** [comment]
When you say factory reset,  did you just click reset everything?  or did you actually do a full upgrade with restore?

**Comment by VMT-Batman** [comment]
> When you say factory reset,  did you just click reset everything?  or did you actually do a full upgrade with restore?
> 
I actually ended up fixing the issue. I did a factory reset and it continued to do it, but after continuously attempting to jailbreak it wouldn't work. For some reason, it worked almost instantly with the wifi off and Siri off as well

**Comment by bagbyte** [comment]
@VMT-Batman thanks, the trick worked for me as well
